### ISSUE TITLE
How to integrate this in ROS

### ISSUE THREAD

**Issue by abhaysjuneja** [devicehive/devicehive-video-analysis#6]

**Comment by igor-panteleev** [comment]
Hi,
Did you mean `Robot Operating System`?
I have no experience with that.
But this demo should work if ROS can run requirements. Especially `tensorflow` and `opencv`

**Comment by Ritesh1991** [comment]
yes it will run with Ros you but I think you need to make node in Ros with communicate with devicehive python client

**Comment by sa7ina** [comment]
You need to transfer the data through ROS Publisher&Subscriber nodes so the master can route the data to the right listener.

Take a look here:
http://wiki.ros.org/ROS/Tutorials/WritingPublisherSubscriber%28python%29
